CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -578,16 +585,23 @@ end
 -- any request raise dialogs.
 --!param amount (integer) The amount, in game dollars per month, to increase
 -- the salary by.
+--!return (bool) Whether the wage actually increased
 function Staff:increaseWage(amount)
-  self.profile.wage = self.profile.wage + amount
-  self.world.ui:playSound("cashreg.wav")
-  if self.profile.wage > 2000 then -- What cap here?
-    self.profile.wage = 2000
-  else -- If the cap has been reached this member of staff won't get unhappy
-       -- ever again...
-    self:changeAttribute("happiness", 0.99)
-    self:setMood("pay_rise", "deactivate")
+  -- Are we already paid the maximum?
+  local success = true
```

Comment:
If you name it `wage_raised` it becomes more clear what `true` means.

Reply:
Forgot this one, it's added now.